server/src/test/kotlin/core/bettinground/BetTest.kt

```diff
@@ -18,7 +18,7 @@ class BetTest {
         val player2 = Player(position = 2, bet = 0, stack = startingStack - blindsMock.bigBlind, lastAction = null)
```

Comment:
Imo writing the values explicitely instead of arithmetic on blind / stack mocks would be more readable, especially since the mocks are not defined it the test's scope and it's not obvious what are the exact values (applies even more to the tests below).

Reply:
sure